### PR TITLE
More detailed addon logging

### PR DESCRIPTION
**Pull request by jcpunk**
It would be handy to have anaconda spit out logs on what all it is loading for addons.

### CODE REVIEW

**Comment by jkonecny12** [comment]
Jenkins, it's ok to test.

File: pyanaconda/addons.py [hunk 1]
```diff
@@ -88,6 +91,7 @@ def execute(self, storage, ksdata, users, payload):
         for v in self.__dict__.values():
             if hasattr(v, "execute"):
                 progress_message(N_("Executing %s addon") % v.name)
+                log.debug("Executing %s addon", v.name)
```
Comment (poncovka) [inline pyanaconda/addons.py]: I think that the progress message above is logged as well, so it is not necessary to log it again.

File: pyanaconda/addons.py [hunk 2]
```diff
@@ -103,6 +107,7 @@ def setup(self, storage, ksdata, payload):
         self.__dict__ = d
         for v in self.__dict__.values():
             if hasattr(v, "setup"):
+                log.debug("Loading %s addon", v.name)
```
Comment (poncovka) [inline pyanaconda/addons.py]: Please, change it to `progress_message(N_("Setting up %s addon") % v.name)`

File: pyanaconda/addons.py (hunk 1 above)
Comment (jcpunk) [reply pyanaconda/addons.py]: Fixing now.

File: pyanaconda/addons.py (hunk 2 above)
Comment (jcpunk) [reply pyanaconda/addons.py]: Fixing now.

**Comment by jkonecny12** [comment]
Tested, everything works as expected.

**Pull request merged**